src/Core/Abstractions/IAuthService.cs

```diff
@@ -25,6 +25,11 @@ public interface IAuthService
         Task<AuthResult> LogInSsoAsync(string code, string codeVerifier, string redirectUrl, string orgId);
         Task<AuthResult> LogInCompleteAsync(string email, string masterPassword, TwoFactorProviderType twoFactorProvider, string twoFactorToken, bool? remember = null);
         Task<AuthResult> LogInTwoFactorAsync(TwoFactorProviderType twoFactorProvider, string twoFactorToken, string captchaToken, bool? remember = null);
+
+        Task<AuthResult> GetLogInPasswordlessRequestsAsync();
+        Task<AuthResult> LogInPasswordlessAcceptAsync();
+        Task<AuthResult> LogInPasswordlessRejectAsync();
```

Comment:
I think it'd be better to have the verb as the prefix to be easier to read so:
`LogInPasswordlessAcceptAsync` => `AceptLogInPasswordlessAsync`
`LogInPasswordlessRejectAsync ` => `RejectLogInPasswordlessAsync`

Reply:
This will probably be merged into one method with different arguments, it will certainly change in the future. But to be coherent with the other methods in the service i'll leave it like this.